corehq/apps/cloudcare/static/cloudcare/js/form_entry/spec/form_ui_spec.js

```diff
@@ -75,13 +75,48 @@ hqDefine("cloudcare/js/form_entry/spec/form_ui_spec", function () {
             assert.equal(form.children()[0].children().length, 1);
             assert.equal(form.children()[0].children()[0].type(), constants.GROUP_TYPE);
             assert.isTrue(form.children()[0].children()[0].isRepetition);
-            assert.equal(form.children()[0].children()[0].children()[0].type(), constants.QUESTION_TYPE);
+            assert.equal(form.children()[0].children()[0].children()[0].type(), constants.GROUPED_QUESTION_TILE_ROW_TYPE);
+            assert.equal(form.children()[0].children()[0].children()[0].children()[0].type(), constants.QUESTION_TYPE);
+        });
+
+        it('Should render questions grouped by row', function () {
+            let styleObj = {raw: '2-per-row'};
+            let q0 = fixtures.textJSON({
+                style: styleObj,
+                ix: "0",
+            });
+            let g0 = fixtures.groupJSON({
+                ix: "1",
+            });
+            g0.children[0].children[0].style = styleObj;
+            g0.children[0].children[0].style = styleObj;
```

Comment:
should this be `g0.children[0].children[1].style = styleObj;`?

Reply:
yes https://github.com/dimagi/commcare-hq/pull/33389